app/cdap/components/Administration/TetheringTabContent/OdfTetheringConnections/PendingRequests/PendingRequestsTable.tsx

```diff
@@ -15,14 +15,147 @@
  */
 
 import React from 'react';
+import T from 'i18n-react';
+import styled, { css } from 'styled-components';
+import { IConnection, IPendingReqsTableData } from '../types';
+import { formatAsPercentage } from 'services/DataFormatter';
+import { humanReadableDate } from 'services/helpers';
 
-const PendingRequestsTable = () => {
-  return (
-    <>
-      {/* TODO: Add table here*/}
-      <span> Table Goes Here </span>
-    </>
-  );
+const PREFIX = 'features.Administration.Tethering';
+const REQUEST_DATE_FORMAT = 'MM/DD/YYYY - hh:mm A';
+
+const PENDING_REQS_TABLE_HEADERS = [
+  {
+    property: 'requestTime', // properties may come handy when adding sort by columns to the table, will remove if not needed
+    label: T.translate(`${PREFIX}.ColumnHeaders.requestTime`),
+  },
+  {
+    property: 'gcp',
+    label: T.translate(`${PREFIX}.ColumnHeaders.gcp`),
+  },
+  {
+    property: 'instanceName',
+    label: T.translate(`${PREFIX}.ColumnHeaders.instanceName`),
+  },
+  {
+    property: 'region',
+    label: T.translate(`${PREFIX}.ColumnHeaders.region`),
+  },
+  {
+    property: 'omniNamespace',
+    label: T.translate(`${PREFIX}.ColumnHeaders.omniNamespace`),
+  },
+  {
+    property: 'cpu',
+    label: T.translate(`${PREFIX}.ColumnHeaders.cpu`),
+  },
+  {
+    property: 'memory',
+    label: T.translate(`${PREFIX}.ColumnHeaders.memory`),
+  },
+  {
+    label: '',
+  },
+];
+
+const GridContainer = styled.div`
```

Comment:
Not in this PR, but we should start using a reusable table/entity listing component. We do have https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/components/shared/Table/index.tsx - did you look at that?

Reply:
Agreed, I was just thinking that we need at least three more tables for tethering that have a lot in common. Will look into the one you linked and see how we can utilize it.